### DISCUSSION TITLE
Microsoft Azure Storage Explorer Icon Please

### DISCUSSION THREAD

**Discussion by neeku** [elrumo/macOS_Big_Sur_icons_replacements#1473] [Icon request]
I would love to have a fresh icon with a different color theme that's not blue+white.

**Comment by itjustcrashed** [comment]
I made this one look similar to the original, but I added a gradient background, rounded the corners, and mac-ified it!
Let me know if you want a different one, I have way too much time on my hands.
https://macosicons.com/#/u/wannafedora
Waiting for approval, but here is the PNG:
![Azure_Storage_Explorer](https://github.com/elrumo/macOS_Big_Sur_icons_replacements/assets/151413500/97fc5424-64e3-435e-814e-848623f85205)
